specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/networkData.json

```diff
@@ -0,0 +1,408 @@
+            },
+            "inboundConnectedResources": {
+              "readOnly": true,
+              "type": "object",
+              "description": "The allowed connections inbound details",
+              "properties": {
+                "tcpPortsSummary": {
+                  "readOnly": true,
+                  "type": "integer",
+                  "description": "The allowed connections inbound tcp ports summary"
+                },
+                "udpPortsSummary": {
+                  "readOnly": true,
+                  "type": "integer",
+                  "description": "The allowed connections inbound udp ports summary"
+                },
+                "tcpPorts": {
+                  "readOnly": true,
+                  "type": "string",
+                  "description": "The allowed connections inbound tcp ports"
+                },
+                "udpPorts": {
+                  "readOnly": true,
+                  "type": "string",
+                  "description": "The allowed connections inbound udp ports"
+                }
+              }
+            },
+            "outboundConnectedResources": {
+              "readOnly": true,
+              "type": "object",
+              "description": "The allowed connections outbound details",
+              "properties": {
+                "tcpPortsSummary": {
+                  "readOnly": true,
+                  "type": "integer",
+                  "description": "The allowed connections outbound tcp ports summary"
+                },
+                "udpPortsSummary": {
+                  "readOnly": true,
+                  "type": "integer",
+                  "description": "The allowed connections outbound udp ports summary"
+                },
+                "tcpPorts": {
+                  "readOnly": true,
+                  "type": "string",
+                  "description": "The allowed connections outbound tcp ports"
+                },
+                "udpPorts": {
+                  "readOnly": true,
+                  "type": "string",
+                  "description": "The allowed connections outbound udp ports"
+                }
+              }
+            }
+          }
+        },
+        "actualTraffic": {
+          "readOnly": true,
+          "type": "object",
+          "description": "The actual traffic details",
+          "properties": {
+            "timeRange": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The actual traffic time range"
+            },
+            "TrafficDataState": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The actual traffic state"
+            },
+            "inboundConnectedResources": {
+              "readOnly": true,
+              "type": "object",
+              "description": "The actual traffic inbound details",
+              "properties": {
+                "actualTrafficVolume": {
```

Comment:
I think that since we are in the "actualTraffic" object you can omit the "actual" workd from all the properties to make the fields a little simpler